cmd/juju/model/exportbundle.go

```diff
@@ -38,12 +39,17 @@ Exports the current model configuration as a reusable bundle.
 If --filename is not used, the configuration is printed to stdout.
  --filename specifies an output file.
 
+If --export-with-series is used, the configuration will include series data
+ alongside bases. This should be used as a compatibility option if you wish your
+ bundle to be compatible with versions of juju before bases were introduced
```

Comment:
missing full stop

Reply:
Nit: juju -> Juju

And I think this sentence is a bit confusing, it could be rephrased more clearly